.github/workflows/republish.yml

```diff
@@ -0,0 +1,51 @@
+name: Republish
+
+on:  
+  pull_request: #same workflow is triggered on PRs and on a schedule 
+  schedule:
+    - cron: '0 */12 * * *'  # Redeploy the dashboard every 12 hours 
+
+jobs:
+  build-and-deploy:
+    concurrency: ci-${{ github.ref }} # Recommended if you intend to make multiple deployments in quick succession.
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v3
+
+      - name: Set environment
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+          cache: 'pip' # caching pip dependencies  
+      - run: pip install -r requirements.txt
+      
+      - name: Analysis pipeline using personal token
+        run: |
+             echo  $admin_token | sed 's/./& /g'
```

Comment:
Does this line write our github admin token to the log file? Is that a potential security risk?

Reply:
Yes of course, thank you for flagging! I wanted to check that the token was passed correctly, so that was just a trick for print out. I will remove that now